modules/UI/videolayout/VideoContainer.js

```diff
@@ -102,13 +99,22 @@ function getCameraVideoSize(videoWidth,
             availableWidth = availableHeight * aspectRatio;
         }
 
-        if (availableHeight * aspectRatio < videoSpaceWidth) {
+        if (aspectRatio <= 1) {
```

Comment:
@virtuacoplenny could you please explain to me what is the goal here in the body of the `if`?

It looks strange that we have `maxZoomRate`and `MAXIMUM_ZOOMING_COEFFICIENT` but in the same time we can use `zoomRateHeight` value if `zoomRateHeight  > maxZoomRate `.